tests/testthat/test-stat_labels.R

```diff
@@ -0,0 +1,124 @@
+test_that("ard_continuous() labels", {
```

Comment:
Rather than having a testing file for this argument, can we move these to a testthat chunk in the respective function test files, e.g. `test-ard_continuous.R`, `test-ard_categorical.R`